src/chunk_insert_state.c

```diff
@@ -221,14 +204,32 @@ tuple_conversion_needed(TupleDesc indesc,
 			indesc->tdhasoid != outdesc->tdhasoid);
 }
 
+/* Translate hypertable indexes to chunk indexes in the arbiter clause */
+static void
+chunk_insert_state_translate_arbiter_indexes(ChunkInsertState *state, ChunkDispatch *dispatch, Relation chunk_rel)
```

Comment:
Optional suggestion: Should we name this `chunk_insert_state_set_arbiter_indexes()` to indicate side-effect (that something is updated/set in the `ChinkInsertState`?